source/includes/gcp/_load_balancer.md

```diff
@@ -18,22 +18,24 @@ curl -X GET \
 {
     "data": [
         {
-            "id": "5538770461112637304",
-            "forwardingRule": "https://www.googleapis.com/compute/v1/projects/test-project/global/forwardingRules/my-forwarding-rule",
-            "targetProxy": "https://www.googleapis.com/compute/v1/projects/test-project/global/targetHttpsProxies/my-target-proxy",
-            "urlMap": "https://www.googleapis.com/compute/v1/projects/test-project/global/urlMaps/my-url",
-            "backendService": "https://www.googleapis.com/compute/v1/projects/test-project/global/backendServices/my-backendServices",
-            "sslCertificates": [
-                "https://www.googleapis.com/compute/v1/projects/test-project/global/sslCertificates/my-ssl-cert"
+            "id": "8268558443601303519",
+            "forwardingRules": [
+                "https://www.googleapis.com/compute/v1/projects/test-area/global/forwardingRules/gfr-test-cme",
+                "https://www.googleapis.com/compute/v1/projects/test-area/global/forwardingRules/gfr-test-qxq"
             ],
-            "name": "my-loadbalancer",
-            "protocol": "HTTPS",
-            "publicAddress": "*.random.net",
-            "backends": [
-                "instance-group-1",
-                "instance-group-2"
+            "targetProxies": [
+                "https://www.googleapis.com/compute/v1/projects/test-area/global/targetHttpProxies/tp-test-fkb",
+                "https://www.googleapis.com/compute/v1/projects/test-area/global/targetHttpsProxies/tp-test-rqi"
+            ],
+            "urlMap": "https://www.googleapis.com/compute/v1/projects/test-area/global/urlMaps/um-test-dfr",
+            "backendServices": [
+                "https://www.googleapis.com/compute/v1/projects/test-area/global/backendServices/be-test-jik"
             ],
-            "shortBackends": "instance-group-1, instance-group-2"
+            "name": "um-test-dfr",
+            "protocol": "HTTPS",
+            "publicAddress": "34.98.108.48, 34.102.228.10",
```

Comment:
This is a list?

Reply:
This is not for the delete operation, it's used in create / list. I will remove

Reply:
Renamed to `publicAddresses`